extension/oidcauthextension/config.go

```diff
@@ -30,3 +30,13 @@ type Config struct {
 	// Optional.
 	GroupsClaim string `mapstructure:"groups_claim"`
 }
+
+func (c *Config) Validate() error {
+	if c.Audience == "" {
+		return errNoAudienceProvided
+	}
+	if c.IssuerURL == "" {
+		return errNoIssuerURL
+	}
+	return nil
+}
```

Comment:
Can be a separate PR?

Reply:
it could - in which case we should do it before this one.

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30460 is filed